app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/NewTransferScreen.kt

```diff
@@ -36,6 +37,10 @@ fun NewTransferScreen(closeActivity: () -> Unit) {
     val navController = rememberNavController()
     var displayConfirmationDialog by rememberSaveable { mutableStateOf(false) }
 
+    BackHandler(!displayConfirmationDialog) {
+        closeActivity()
+    }
```

Comment:
Check if currentDestination is the import screen or else this will trigger for any screen that doesn't override the BackHandler